packages/superset-ui-plugins-demo/storybook/stories/plugin-chart-table/Stories.jsx

```diff
@@ -0,0 +1,110 @@
+/* eslint-disable no-magic-numbers */
+import React from 'react';
+import { SuperChart } from '@superset-ui/chart';
+import dataLegacy from './dataLegacy';
+import data from './data';
+
+export default [
+  {
+    renderStory: () => (
+      <SuperChart
+        chartType="table2-legacy"
+        chartProps={{
```

Comment:
Check the other examples are updated to use new `SuperChart` api that spread out the fields as top-level props instead of nesting under `chartProps`